docs/essentials/backgrounds.mdx

```diff
@@ -197,8 +246,28 @@ Default: `0.5`
 
 The opacity of the grid lines.
 
+<Callout variant="warning" icon="🆕">
+
+  **With the [`globals` API](#globals-api)**
+
+  #### `options`
+
+  Type: `Record<string, { name: string; value: string }>`
+
+  Required: See description
+
+  Defines the background colors. Replaces [`values`](#values). See above for a [usage example](#configuration).
+
+  Required when defining `backgrounds` parameter at project level.
```

Comment:
There's a typo here, as demonstrated by my suggestion, and the same issue is present below: users can define this at the component and story levels if they want.
```suggestion
  Required when defining the `backgrounds` parameter at the project level.
```